src/App.js

```diff
@@ -5,6 +5,7 @@ import './App.css';
 class App extends React.Component {
   constructor(props) {
     super(props);
+
     this.state = {};
```

Comment:
* [OPTIONAL]
*  This is an optional suggestion 
* You can use class components without a state by extending the class using `React.PureComponent`  instead of `React.Component` Example : `import React, { PureComponent } from 'react'; `

` class Calculator extends PureComponent { // or React.PureComponent ...`  [Learn more ](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md)